src/SFA.DAS.EmployerIncentives.Web/Services/Applications/IApplicationService.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using SFA.DAS.EmployerIncentives.Web.ViewModels.Apply;
+
+namespace SFA.DAS.EmployerIncentives.Web.Services.Applications
+{
+    public interface IApplicationService
+    {
+        Task<Guid> Post(string accountId, string accountLegalEntityId, IEnumerable<string> apprenticeshipIds);
```

Comment:
Should this be names Create?  POST is how it's implemented rather than what it does.

Reply:
ok by me. Wasn't sure of intentions of these services they appear to a mix between a client and a service. So I assumed naming was in a client style